tests/clients_last_seen_v1/test_single_day/clients_daily_v6.schema.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "submission_date_s3",
+    "type": "DATE",
+    "mode": "REQUIRED",
+    "description": "time_partitioning_field"
+  },
+  {
+    "name": "active_hours_sum",
+    "type": "FLOAT",
+    "mode": "REQUIRED"
+  },
+  {
+    "name": "attribution",
+    "type": "RECORD",
+    "mode": "REQUIRED",
+    "fields": [
+      {
+        "name": "source",
+        "type": "STRING",
+        "mode": "REQUIRED"
+      }
+    ]
+  },
+  {
+    "name": "client_id",
+    "type": "STRING",
+    "mode": "REQUIRED"
+  }
+]
```

Comment:
I made this support `yaml` and `ndjson` schemas as well, but `json` is supported by the `bq` cli tool, so I'm not sure if it's better to stick with `json` even though `yaml` is easier to maintain.

Reply:
My vote is to stick with json for this due to it being an existing standard that interoperates with `bq`.